Lib/ufo2ft/fontInfoData.py

```diff
@@ -311,8 +311,8 @@ def postscriptWeightNameFallback(info):
     openTypeVheaCaretOffset=None,
 
     postscriptUniqueID=None,
-    postscriptUnderlineThickness=None,
-    postscriptUnderlinePosition=None,
+    postscriptUnderlineThickness=50,
+    postscriptUnderlinePosition=-75,
```

Comment:
What I meant was that, IIUC, AFDKO assumes that upem=1000.  For our pipeline to work with arbitrary upem's, we should scale these.  Ie, instead of 50, use int(round(50 * upem / 1000))